src/main/java/codeu/model/store/persistence/PersistentDataStore.java

```diff
@@ -102,7 +102,14 @@ public PersistentDataStore() {
         UUID ownerUuid = UUID.fromString((String) entity.getProperty("owner_uuid"));
         String title = (String) entity.getProperty("title");
         Instant creationTime = Instant.parse((String) entity.getProperty("creation_time"));
-        Conversation conversation = new Conversation(uuid, ownerUuid, title, creationTime);
+        // This exists because this is a new feature, so conversations created before
+        // I implemented this were not stored with a value for is_public,
+        // so this makes the default to be true
+        boolean isPublic = true;
+        if (entity.getProperty("is_public") != null) {
```

Comment:
Great job thinking of backwards compatibility

Reply:
Let's add a test case to the PersistentDataStoreTest where we have an entity that doesn't have this is_public property set.

Reply:
I'm not really sure how to create a Conversation where that isn't set, because whenever I instantiate a Conversation now I need to pass in a boolean value in the constructor and I can't pass in null.